src/status_im/ios/core.cljs

```diff
@@ -64,7 +64,8 @@
                        (fn [e]
                          (let [h (.. e -endCoordinates -height)]
                            (when-not (= h @keyboard-height)
-                             (dispatch [:set :keyboard-height h])))))
+                             (dispatch [:set :keyboard-height h])
+                             (dispatch [:set :keyboard-max-height h])))))
```

Comment:
No. 
`:keyboard-height` is the actual height and it can be 0 if the keyboard is hidden.